v3/src/components/tool-shelf/tool-shelf.tsx

```diff
@@ -0,0 +1,103 @@
+import React from "react"
+import {Box, Flex, HStack, Tag, useToast, VStack} from "@chakra-ui/react"
+
+import './tool-shelf.scss'
+import GraphIcon from '../../assets/icons/icon-graph.svg'
+import TableIcon from '../../assets/icons/icon-table.svg'
+import MapIcon from '../../assets/icons/icon-map.svg'
+import SliderIcon from '../../assets/icons/icon-slider.svg'
+import CalcIcon from '../../assets/icons/icon-calc.svg'
+import TextIcon from '../../assets/icons/icon-text.svg'
+import PluginsIcon from '../../assets/icons/icon-plug.svg'
```

Comment:
Move these up with the main body of imports, because they're actually importing React components (after conversion by SVGR).